AmericanMedicalAssociation/ama-style-guide-2
# EWL-11025: reverse sort arrows

**Jira Ticket**
- [EWL-11025: Sort Arrows](https://issues.ama-assn.org/browse/EWL-11025)


## Description
set asc and desc arrows properly


## To Test
- pull and serve
- see instructions in d8 PR https://github.com/AmericanMedicalAssociation/ama-d8/pull/5019


## Visual Regressions
N/A


## Relevant Screenshots/GIFs
N/A


## Remaining Tasks
N/A


## Additional Notes
Anything more to add?

---

[Guidelines for Contribution](CONTRIBUTING.md)
[SG2 Standards](ama-style-guide-2/docs/standards.md)
